src/theory/quantifiers/term_database_sygus.h

```diff
@@ -17,6 +17,8 @@
 #ifndef __CVC4__THEORY__QUANTIFIERS__TERM_DATABASE_SYGUS_H
 #define __CVC4__THEORY__QUANTIFIERS__TERM_DATABASE_SYGUS_H
 
+#include <unordered_map>
```

Comment:
You meant to include unordered_set since you added d_eval_processed as std::unordered_set right?

Reply:
Yes, I've fixed this in my latest commit.